https://gitlab.gnome.org/GNOME/libgweather/-/merge_requests/176

From 98b505675f631c1fabbbcbcee2523f8dde8e7e3d Mon Sep 17 00:00:00 2001
From: Matt Turner <mattst88@gmail.com>
Date: Sun, 20 Mar 2022 12:19:59 -0700
Subject: [PATCH] meson: Use dependency() for finding vapigen

This allows the vapigen binary to be chosen by overriding the VAPIGEN
environment variable.
---
 libgweather/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libgweather/meson.build b/libgweather/meson.build
index 08311af4..7148f5ca 100644
--- a/libgweather/meson.build
+++ b/libgweather/meson.build
@@ -211,7 +211,7 @@ g_ir_scanner = find_program('g-ir-scanner', required: get_option('introspection'
 build_gir = get_option('introspection') and g_ir_scanner.found() and not meson.is_cross_build()

 enable_vala = get_option('enable_vala')
-vapigen = find_program('vapigen', required: enable_vala == 'true')
+vapigen = dependency('vapigen', required: enable_vala == 'true')
 if enable_vala == 'auto' or enable_vala == 'true'
   build_vapi = vapigen.found() and get_option('introspection')
   if enable_vala == 'true' and not build_vapi
--
2.34.1

